qabelbox/build.gradle

```diff
@@ -29,7 +29,7 @@ android {
         applicationId "de.qabel.qabelbox"
         minSdkVersion 19
         targetSdkVersion 22
-        versionCode 1
+        versionCode System.getenv("BUILD_NUMBER") as Integer ?: 1
         versionName "1.0"
```

Comment:
pls set versioname to < 1.0, e.g.
"0.1.0-internal" or 0.1.<BUILDNUMBER>-internal